src/App.php

```diff
@@ -1158,4 +1169,31 @@ private function getRenderedPortals(): array
 
         return $portals;
     }
+
+    protected function emitResponse(): void
+    {
+        $http_line = sprintf('HTTP/%s %s %s',
+            $this->response->getProtocolVersion(),
+            $this->response->getStatusCode(),
+            $this->response->getReasonPhrase()
+        );
+        header($http_line, true, $this->response->getStatusCode());
+        foreach ($this->response->getHeaders() as $name => $values) {
+            foreach ($values as $value) {
+                header("$name: $value", false);
```

Comment:
```suggestion
                header($name: ' . $value, false);
```

Reply:
Ok, but probably in this way : `header($name . ': ' . $value, false);`

Reply:
yes ;-)